src/core/aztec/detector/Detector.ts

```diff
@@ -143,7 +143,7 @@ export default class Detector {
             this.sampleLine(bullsEyeCorners[0], bullsEyeCorners[1], length), // Right side
             this.sampleLine(bullsEyeCorners[1], bullsEyeCorners[2], length), // Bottom
             this.sampleLine(bullsEyeCorners[2], bullsEyeCorners[3], length), // Left side
-            this.sampleLine(bullsEyeCorners[3], bullsEyeCorners[0], length), // Top
+            this.sampleLine(bullsEyeCorners[3], bullsEyeCorners[0], length)  // Top
```

Comment:
Trailing comma should be used, please don't bother removing it and please forgive me if and linter rules says the opposite, in that case they would be wrong.